var/spack/repos/builtin/packages/curl/package.py

```diff
@@ -139,6 +139,10 @@ def determine_variants(cls, exes, version):
             # TODO: Determine more variants.
             return variants
 
+    @property
+    def command(self):
+        return Executable(self.prefix.bin.join('curl-config'))
```

Comment:
This LGTM from Spack perspective, but I wonder if we should also expose `curl` in some way. Maybe we should think of turning "command" into "commands"?